napari/utils/theme.py

```diff
@@ -135,7 +134,7 @@ def increase(font_size: str, pt: int) -> str:
     return f'{int(font_size[:-2]) + int(pt)}pt'
 
 
-def _parse_color_as_rgb(color: str | Color) -> tuple[int, int, int]:
+def _parse_color_as_rgb(color: Union[str | Color]) -> tuple[int, int, int]:
```

Comment:
We're 3.10 and greater, so the Union isn't needed anymore right?
```suggestion
def _parse_color_as_rgb(color: str | Color) -> tuple[int, int, int]:
```

Reply:
And then remove the import.